docs/.vuepress/tools/build.mjs

```diff
@@ -2,7 +2,8 @@ import path from 'path';
 import { fileURLToPath } from 'url';
 import fse from 'fs-extra';
 import utils from './utils.js';
-import { getVersions } from '../helpers.js';
+import { getDocsNonFrameworkedVersions, getDocsFrameworkedVersions, getFrameworks, getLatestVersion,
```

Comment:
This would probably be more readable if each imported prop would be in its own line?